modules/apps/commerce/commerce-frontend-js/test/dev/components/QuantitySelector.js

```diff
@@ -17,12 +17,14 @@ import launcher from '../../../src/main/resources/META-INF/resources/components/
 import '../../../src/main/resources/META-INF/resources/styles/main.scss';
 
 launcher('quantity-selector', 'quantity-selector', {
-	inputName: 'test-name',
-	orderQuantity: 1,
-	settings: {
-		maxQuantity: 29,
-		minQuantity: 1,
-	},
-	size: 'large',
+	onUpdate: (quantity) =>
+		new Promise((resolve) => {
+			setTimeout(() => {
+				// eslint-disable-next-line no-console
+				console.log('The new quantity is:', quantity);
```

Comment:
Please don't log this (see [the guidelines on logging](https://github.com/liferay/liferay-frontend-guidelines/blob/master/dxp/debugging.md#logging)). If you _must_ log it, guard it behind an `if (process.env.NODE_ENV === 'development')` check.

Reply:
This component "starter" won't ever run in Liferay, it lives in our static environment where components are rendered out of context in a blank page. Is this a margin that's reasonable enough to leave this `console.log` where it is? 😄  If not, I'll figure out something else 👍

Reply:
Ah, I didn't even notice the path. No need to worry about the `NODE_ENV` check then, but I think you should just get rid of the `console.log` because lint suppressions _are_ a smell. The more we normalize them, the more people get used to seeing them and before you know it the repo is littered with `console.log` and suppressions and we have a mess.

Reply:
Cool, cool.